emails/models.py

```diff
@@ -495,6 +494,7 @@ class RelayAddress(models.Model):
     num_spam = models.PositiveIntegerField(default=0)
     generated_for = models.CharField(max_length=255, blank=True)
     block_list_emails = models.BooleanField(default=False)
+    used_on = models.CharField(default=None, blank=True, null=True, max_length=255)
```

Comment:
Note: this will hold a comma-separated list of domains. So the `max_length` of 255 might eventually be a problem if someone uses the same alias on enough domains. :/